definitions/infra-kubernetespod/definition.yml

```diff
@@ -0,0 +1,38 @@
+domain: INFRA
+type: KUBERNETES_POD
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+
+synthesis:
+  identifier: entity.id
+  name: k8s.podName
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: k8s.pod.isReady
+  tags:
+    k8s.status:
```

Comment:
Should this add aliasing? (same for `isReady` and `isScheduled` below)

Reply:
This doesn't need aliasing here. Only the k8s container will need both tags `k8s.status` and `container.state`.

Reply:
@srvaroa is this yaml already supporting more than 1 rule per attribute so that we can achieve what I mention in the comment above?

Reply:
Indeed, it supports it, you can define different rules with their specific tags for a single entity, container is a good example https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/infra-container/definition.yml#L32

Reply:
So I can do this?

```yaml
k8s.status:
        entityTagName: container.state
k8s.status:
```

And I would get the `k8s.status` and `container.state` tags?

Reply:
Not exactly since`k8s.status` would be a duplicated map key but if you are doing this you probably have 2 sources of telemetry and want to have two rules, so you could do something like this:

```
rules:
  - identifier: attributeA
  ....
  tags:
  k8s.status:
    entityTagName: container.state
 - identifier: attributeA
   conditions:
      .....
  ....
  tags:
  k8s.status
```